src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -274,11 +281,11 @@ void EnAnubis_ShootFireball(EnAnubice* this, GlobalContext* globalCtx) {
         Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 5, 3000, 0);
     }
 
-    EnAnubice_SetFireballRot(this, globalCtx);
+    EnAnubice_SetHeadRot(this, globalCtx);
```

Comment:
we can revert the func name too
SetFireBallDirection or something along those lines might make more sense over just plainly setting the rotation. Or involve that its calculating where to send the fireball to hit link

Reply:
AimFireball?

Reply:
possibly. that makes it sound like more of an action to me than just a simple calculation + set, but its probably fine